config/locales/en.yml

```diff
@@ -801,11 +802,12 @@ en:
       title: Security Events
       description_html: "This page shows the security events that have occurred on %{gem}. If you see any suspicious activity, please <a href='mailto:support@rubygems.org'>contact support</a>."
     transfer:
+      cancelled: Your draft gem transfer has been cancelled.
       organizations:
         new:
           organization: Organization
       confirm:
-        success: "%{gem} has been transferred successfully to %{organization}."
+        success: "Your gems have been transferred successfully to %{organization}."
```

Comment:
💅 "Your" here is definitely correct in context because we're enforcing ownership of transferred gems, but I think omitting "your" is just as valid and wouldn't need to change if our requirements changed.